config/responsecache.php

```diff
@@ -34,4 +34,9 @@
      * configured in app/config/cache.php
      */
     'cache_store' => env('RESPONSE_CACHE_DRIVER', 'file'),
+
+    /*
+     * If selected cache driver supports tagging you can define it here.
```

Comment:
Something like this? 

You would use tags to separate cached data when sharing same storage. This way you will be able to flush response cache and keep your application data.